modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/paths/APIBuilder.path

```diff
@@ -135,6 +135,21 @@
 	<td>//button[(@class='property-container btn btn-unstyled') and contains(.,'${key_text}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>DISABLED_PROPERTY_CONTAINER</td>
+	<td>//button[(@class='property-container btn btn-unstyled') and contains(.,'${key_text}')]//div[contains(@class,'disabled')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DELETE_BUTTON_FROM_PROPERTY_BOX</td>
+	<td>//div[@class='autofit-row' and contains(.,'${key_commentValue}')]//button[contains(@aria-label, 'Delete')]</td>
+	<td></td>
```

Comment:
you can try to use 'Icon#ANY' to replace it, could you take a look at it? thanks!

Reply:
@AkramYoussoufi does Icon#ANY works for you? If yes, you can remove the DELETE_BUTTON_FROM_PROPERTY_BOX

Reply:
with Icon#ANY you also can pass a text key to determine. Example:
```
Click(
				key_text = "expand",
				locator1 = "Icon#ANY");
```

Reply:
yes this works 

```
Click(
				key_text = "trash",
				locator1 = "Icon#ANY");
```

but it doesn't select the right proprieties element to delete from the box properties it is always select the first properties element because a lot of element has the key_text="trash" the same, it's basically not specific to what to delete inside of the property box.

Reply:
if we have multiple elements inside of the property box and you wish to delete 'Create Date' as described on the test it will eventually select the first element within the properties box because all elements within it have the trash icon.

Reply:
ok, gotcha! thanks for checking ✅